webpack/scenes/AlternateContentSources/Create/Steps/SelectSource.js

```diff
@@ -68,23 +99,34 @@ const SelectSource = () => {
         >
           <FormSelect
             isRequired
+            isDisabled={acsType === 'rhui'}
             value={contentType}
             onChange={(value) => {
               setContentType(value);
             }}
             aria-label="FormSelect Input"
           >
             {
-                            typeOptions.map(option => (
-                              <FormSelectOption
-                                key={option.value}
-                                value={option.value}
-                                label={option.label}
-                              />
-                            ))
-                        }
+              typeOptions.map(option => (
+                <FormSelectOption
+                  key={option.value}
+                  value={option.value}
+                  label={option.label}
+                />
+              ))
+            }
           </FormSelect>
         </FormGroup>
+        {acsType === 'rhui' &&
+        <>
+          <Alert variant="info" title={__('Generate RHUI certificates for the desired repositories as necessary.')} />
+          <ClipboardCopy hoverTip="Copy" clickTip="Copied" variant="inline-compact" isBlock>
+            rhui-manager client cert --name rhui-acs-certs --days 365 --dir /root
+            --repo_label rhui-repo-1,rhui-repo-2
+          </ClipboardCopy>
+          {__('Create content credentials with the generated SSL certificate and key via Content -> Content Credentials.')}
```

Comment:
From our meeting, the main improvement suggested was adding a link here. What do you think about having the following text be an entire link?

"Create content credentials with the generated SSL certificated and key."

Reply:
Good idea.